example2/js/vendor/leia/shaders/FragmentShader.js

```diff
@@ -0,0 +1,159 @@
+export var FRAG_SHADER = `precision highp float;
+precision highp int;
+
+// our textures
+uniform sampler2D _texture_0;
+uniform sampler2D _texture_1;
+uniform sampler2D _texture_2;
+uniform sampler2D _texture_3;
+uniform sampler2D _texture_4;
+uniform sampler2D _texture_5;
+uniform sampler2D _texture_6;
+uniform sampler2D _texture_7;
+
+// other inputs
+uniform float _width;// = 2560.0;
+uniform float _height;// = 1600.0;
+
+uniform float _viewResX;// = 640.0;
+uniform float _viewResY;// = 400.0;
+
+uniform float _viewsX;//= 4.0;
+uniform float _viewsY;// = 1.0;
+
+uniform mat4 _interlace_matrix; /* = mat4(
```

Comment:
Should we remove the commented out mat4s?

Reply:
Sure.